dist/ui/Editor.js.flow

```diff
@@ -205,7 +205,9 @@ class Editor extends React.PureComponent<any, any, any> {
   }
 
   componentWillUnmount(): void {
+    this._autoFocusTimer && clearTimeout(this._autoFocusTimer);
     this._editorView && this._editorView.destroy();
+    this._editorView && (this._editorView._props = {});
```

Comment:
I don't understand this.
could you please explain what `_props` is for?

Reply:
It's in `EditorView` in `prosemirror-view` https://github.com/ProseMirror/prosemirror-view/blob/master/src/index.js#L27 I saw in memory profiling that this was one of the larger items sticking around and not getting garbage collected, and cleaning it up also helped with garbage collecting of other instances in our code.